muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/aside.tsx

```diff
@@ -36,11 +36,11 @@ class Navigation extends React.Component<NavigationProps, NavigationState> {
   isVisible(hash: string) {
     switch (hash) {
       case "hops":
-        return this.props.status.isActiveUser && !this.props.records.studyEndDate && this.props.hops.eligibility && this.props.hops.eligibility.upperSecondarySchoolCurriculum === true;
+        return this.props.status.isActiveUser && this.props.hops.eligibility && this.props.hops.eligibility.upperSecondarySchoolCurriculum === true;
```

Comment:
Is it necessary to check upperSecondarySchoolCurriculum with `===`  as its type is already boolean, so we can only expect true or false value? So ` && this.props.hops.eligibility.upperSecondarySchoolCurriculum` should be enough. Might be unnecessary nitpicking. Of course the original is more stricter if we can expected value be something else than boolean also.